src/module/classes/barbarian.mjs

```
@@ -1,32 +1,17 @@
-        improvedFeral(actor,combatants)
-    }else{
-        
-        // console.log("Nothing ")
+export function feral(actor) {
+    const rageFeature = actor.items.find(i => i.name === "Rage");
+    let notification = "elkan5e.notifications.FeralInstincts";
+    if (rageFeature && (actor.items.find(i => i.name === "Feral Instincts") || actor.items.find(i => i.name === "Improved Feral Instincts"))) {
+        if (actor.items.find(i => i.name === "Improved Feral Instincts")) {
+            notification = "elkan5e.notifications.ImprovedFeralInstincts";
+        }
+        let uses = rageFeature.system.uses.max - rageFeature.system.uses.spent;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'uses' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)